app/controllers/podcast_planner_controller.rb

```diff
@@ -34,21 +34,51 @@ def set_beginning_of_week
     Date.beginning_of_week = :sunday
   end
 
-  # Only allow a list of trusted parameters through.
+  # Make params match PodcastPlanner interface
```

Comment:
Here's the first bit of magic.  I swapped several fields out, and instead "infer" them here.  Translating into the existing PodcastPlanner interface.